### ISSUE TITLE
Add banner to user and developer facing AMO pages

### ISSUE THREAD

**Issue by caitmuenster** [mozilla/addons#992]
Until we have a fix for the certificate issue, let's put up a banner on all AMO pages that face users and developers that says: 

.You may be experiencing issues with extensions and themes for Firefox. Please see https://discourse.mozilla.org/t/certificate-issue-causing-add-ons-to-be-disabled-or-fail-to-install/39047 for more information."

**Comment by AdrianoCahete** [comment]
And [pin a issue here at GH](https://help.github.com/en/articles/pinning-an-issue-to-your-repository), otherwise someone will need to close countless issues with the same problem.

**Comment by willdurand** [comment]
@caitmuenster I don't think this is something we can do without deploying a new version, which is risky...

**Comment by diox** [comment]
We've deployed a banner for developers specifically since approvals are currently disabled.

**Comment by kumar303** [comment]
Once we have a backend, we could reuse this UI for showing banners: https://github.com/mozilla/addons-frontend/pull/5620
